koinos/chain/system_calls.proto

```diff
@@ -208,22 +208,22 @@ message consume_block_resources_result {
 ///////////////////////////////////////////////////////////////////////////////
 
 message put_object_arguments {
-   object_space space = 1;
```

Comment:
With the duplicate `object_space`, we should continue using the `chain` version elsewhere for now to preserve backwards compatibility.